index.html

```diff
@@ -1,13 +1,232 @@
+                <span class="info-text">2015</span></li>
+            </ul>
+          <p class="project-description">A daily selection of privately personalized reads; no accounts or sign-ups required</p>
+        <ul class="tech-list">
+          <li class="project-tech">html</li>
+          <li class="project-tech">css</li>
+          <li class="project-tech">javascript</li>
+        </ul>
+          <button class="outlined-button">See Project</button>
+      
+      </div>
+  </div>
+
+</div>
+</section>
+
+<section class="about">
+      <div class="about-intro-text">
+          <p class="project-name h-text">About Myself</p>
+          <p class="project-description test4">
+              I'm a software developer! I can help you build a product, feature or website. Look through some of
+              my work and experience! If you like what you see and have a project you need coded, don't hesitate
+              to contact me.
+          </p>
+          <span class="light-purple letsconnect">LET'S CONNECT</span>
+          <div class="about-icons">
+              <i class="ri-twitter-fill"></i>
+              <i class="ri-linkedin-fill"></i>
+              <i class="ri-medium-line"></i>
+              <i class="ri-github-fill"></i>
+          </div>
+          <button class="outlined-button btn">Get my resume</button>
+      </div>
+      <ul class="d-flex about-skills-container">
+        <li class="about-skill-container">
+            <div class="d-flex about-skill-heading">
+                <h4 class="about-heading-small">Languages</h4>
+                <i class="fa-solid fa-angle-down"></i>
+            </div>
+            <div class="about-skill-body-container">
+                <div class="d-flex about-skill-body">
+                    <img class="about-skill-body-img" src="./assets/jslogo.svg" alt="js">
+                    <h5 class="about-skill-text">Javascript</h5>
+                </div>
+                <div class="d-flex about-skill-body">
+                    <img class="about-skill-body-img" src="./assets/html5logo.svg" alt="html">
+                    <h5 class="about-skill-text">HTML</h5>
+                </div>
+                <div class="d-flex about-skill-body">
+                    <img class="about-skill-body-img" src="./assets/csslogo.svg" alt="css">
+                    <h5 class="about-skill-text">CSS</h5>
+                </div>
+            </div>
+        </li>
+        <li class="about-skill-container">
+            <div class="d-flex about-skill-heading">
+                <h4 class="about-heading-small">Frameworks</h4>
+                <i class="fa-solid fa-angle-right"></i>
+            </div>
+        </li>
+        <hr>
+        <li class="about-skill-container">
+            <div class="d-flex about-skill-heading">
+                <h4 class="about-heading-small">Skills</h4>
+                <i class="fa-solid fa-angle-right"></i>
+            </div>
+        </li>
+        <hr>
+    </ul>
+    </section>
```

Comment:
- [ ] Like I pointed on your project cards, the design are great but they missing some tiny bits but important details to make your design really as expected and to show your great knack for smart designs. Some `padding`s are too much, making the design a little unpleasant while fonts like the `About Me` don't have the correct value, Please kindly compare and make them match :point_down: 

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/219875813-e8552019-18ad-4957-b0d9-89e07793674a.png)|![image](https://user-images.githubusercontent.com/99715640/219875828-52945d94-aa3b-4077-bb12-907f82d3f10a.png)|